imports/plugins/included/product-detail-simple/client/containers/productDetail.js

```diff
@@ -272,7 +272,7 @@ function composer(props, onData) {
   const tagSub = Meteor.subscribe("Tags");
   const shopIdOrSlug = Reaction.Router.getParam("shopSlug");
   const productId = Reaction.Router.getParam("handle");
-  const variantId = Reaction.Router.getParam("variantId");
+  const variantId = ReactionProduct.selectedVariant()._id;
```

Comment:
You could probably use `ReactonProduct.selectedVariantId()` instead.

Reply:
Thanks I just refactored it to `selectedVariantId()`